src/i_video.c

```diff
@@ -1656,8 +1671,8 @@ static void CreateSurfaces(int w, int h)
 
     I_InitDiskFlash();
 
-    SDL_SetWindowMinimumSize(screen, video.unscaledw * 2,
-                             use_aspect ? ACTUALHEIGHT * 2 : SCREENHEIGHT * 2);
+    SDL_SetWindowMinimumSize(screen, video.unscaledw,
+                             use_aspect ? ACTUALHEIGHT : SCREENHEIGHT);
```

Comment:
Should we leave the minimum size at 400p for `scalefactor == 0`? I don't like it when I accidentally resize the window and make it too small.

Reply:
Yes, good idea.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1644/commits/7201533f13714ce4e1ab2c52a41081d4ab051275